From fc76f8dbee8e6f32d111ef7dc84dc43bebcbd4ac Mon Sep 17 00:00:00 2001
From: Meng Li <Meng.Li@windriver.com>
Date: Mon, 8 Feb 2021 14:47:30 +0800
Subject: [PATCH] bcm-2xxx-rpi: fix build issue

physvirt_offset and vmemmap had beed removed by upstream
commit 7bc1a0f9e176("arm64: mm: use single quantity to represent
the P to VA translation"). But it was involved into branch
v5.4/standard/bcm-2xxx-rpi again when merge code. So, remove them
to fix the building error.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/mm/init.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm64/mm/init.c b/arch/arm64/mm/init.c
index cb941cb24924..53037b6b6708 100644
--- a/arch/arm64/mm/init.c
+++ b/arch/arm64/mm/init.c
@@ -53,12 +53,6 @@
 s64 memstart_addr __ro_after_init = -1;
 EXPORT_SYMBOL(memstart_addr);
 
-s64 physvirt_offset __ro_after_init;
-EXPORT_SYMBOL(physvirt_offset);
-
-struct page *vmemmap __ro_after_init;
-EXPORT_SYMBOL(vmemmap);
-
 /*
  * We create both ZONE_DMA and ZONE_DMA32. ZONE_DMA covers the first 1G of
  * memory as some devices, namely the Raspberry Pi 4, have peripherals with
-- 
2.17.1

